addons/chemical/CfgVehicles.hpp

```diff
@@ -145,15 +145,15 @@ class CfgVehicles {
                 displayName = CSTRING(UI_gasType);
                 typeName = "NUMBER";
                 class values {
+                    class TearGas {
+                        name = CSTRING(Lvl0_Gas);
+                        value = 0;
+                        default = 1;
```

Comment:
I think it would make sense to keep ToxicGas as default (remove the "default = 1; here and put it into the ToxicGas class)